### PR TITLE
gsar 1.21 (new formula)

### PR DESCRIPTION
**Pull request by vszakats**

### CODE REVIEW

File: Library/Formula/gsar.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Gsar < Formula
+  desc "General Search And Replace on files"
+  homepage "http://tjaberg.com/"
+  url "http://tjaberg.com/gsar121.zip"
+  sha256 "05fb9583c970aba4eb0ffae2763d7482b0697c65fda1632a247a0153d7db65a9"
+  version "1.21"
```
Comment (bfontaine) [inline Library/Formula/gsar.rb]: Please move this above the checksum.

File: Library/Formula/gsar.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  def install
+    system "make"
+    bin.install "gsar"
```
Comment (bfontaine) [inline Library/Formula/gsar.rb]: Is there no `make install` we could use?

Comment (vszakats) [inline Library/Formula/gsar.rb]: No, unfortunately it's missing from the Makefile.

File: Library/Formula/gsar.rb (hunk 1 above)
Comment (vszakats) [inline Library/Formula/gsar.rb]: Yes, got those, and the whitespace problems as well. Thanks!

**Comment by bfontaine** [comment]
Is it a popular project? It’s hard to find anything on Google.

**Comment by vszakats** [comment]
I don't have any information on it's popularity. I've been using it for 10+ years on the Windows platform and I've recently found references for it on Stackoverflow. It's a very practical and fast simple search and replace tool for those occasions when `ack` and `sed` is "too much". Having this on OS X also helps making the shift to the platform without having to rewrite much existing script logic.

**Comment by bfontaine** [comment]
Thank you @vszakats! :smile:

**Pull request closed without merging**